src/components/Calculator.js

```diff
@@ -1,21 +1,39 @@
-import React, { PureComponent } from 'react';
+/* eslint-disable no-mixed-spaces-and-tabs */
+/* eslint-disable no-tabs */
+/* eslint-disable react/no-unused-state */
```

Comment:
- [ ] Please, consider removing all linter disables, to make the linter do its work correctly and avoid future bugs in your project, find them now and solve them in time 👍